prow/config.yaml

```diff
@@ -1481,6 +1538,40 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  - name: pull-kubernetes-bazel-test
+    agent: kubernetes
+    context: pull-kubernetes-bazel-test
+    always_run: true
+    rerun_command: "/test pull-kubernetes-bazel-test"
+    trigger: "(?m)^/test( all| pull-kubernetes-bazel-test),?(\\s+|$)"
+    branches:
+    - release-1.10
+    labels:
+      preset-service-account: true
+      preset-bazel-scratch-dir: true
+      preset-bazel-remote-cache-enabled: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180418-79a97ca02-1.10
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=60"
+        - "--" # end bootstrap args, scenario args below
+        - "--test=//... -//build/... -//vendor/..."
+        - "--manual-test=//hack:verify-all"
+        - "--test-args=--config=unit"
+        - "--test-args=--build_tag_filters=-e2e,-integration"
+        - "--test-args=--test_tag_filters=-e2e,-integration"
+        - "--test-args=--flaky_test_attempts=3"
+        # Bazel needs privileged mode in order to sandbox builds.
```

Comment:
side note: can this be put into a pod preset?

Reply:
Nope. They can do env, volumes, volume mounts, and are keyed by labels.

Reply:
We implemented what k8s has which seems to be targeted at credential injection only.